code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -101,6 +105,10 @@
 		real_name = name
 	if(!loc)
 		stack_trace("Simple animal being instantiated in nullspace")
+	NPC_count += 1
+	if(NPC_count > MAX_NPCs)
+		qdel(src)
+		visible_message("<span class='danger'>Die, monster! You don't belong in this world!</span>")
```

Comment:
Does this need to be a visible message to everyone? It's kind of an OOC deletion.

Reply:
Only admins get meme humor now.